README.rst

```diff
@@ -17,9 +17,7 @@ The custom ops used by Pointnet++ are currently **ONLY** supported on the GPU us
 Setup
 -----
 
-* Install ``python3.6`` -- This repo is currently only tested with/officially supports ``python3.6``
-
-  All further instructions assume that `python` defaults to ``python3.6``
+* Install ``python3.6``
```

Comment:
```suggestion
* Install ``python`` -- This repo is tested with ``2.7``, ``3.4``, ``3.5``, and ``3.6``
```

Reply:
3.4 was definitely a typo because torch 1.0.0 doesn't support that far back. I must have tested on 2.7 and then 3.6 twice. :/ 3.5 builds and passes tests now, and I've removed 3.4 from the readme and tox.ini.